website/project/model.py

```diff
@@ -3308,6 +3297,16 @@ def get_admin_contributors_recursive(self, unique_users=False, *args, **kwargs):
                     else:
                         yield (contrib, node)
 
+    def get_admin_contributors(self, users):
+        """Return a set of all admin contributors for this node and admin
+        contributors on descendant nodes. Excludes contributors on node links and
+        inactive users.
+        """
+        return [
+            user for user in users
+            if self.has_permission(user, 'admin') and
+            user.is_registered]
```

Comment:
What about merged and disabled users? Should this check for `user.is_active` instead?